docs/zh_cn/recommended_topics/deploy/mmdeploy_guide.md

```diff
@@ -271,7 +403,7 @@ python3 ${MMDEPLOY_DIR}/tools/test.py \
 - `--model`: 导出的后端模型。 例如, 如果我们导出了 TensorRT 模型，我们需要传入后缀为 ".engine" 文件路径。
 - `--out`:  保存 pickle 格式的输出结果，仅当您传入这个参数时启用。
 - `--format-only`: 是否格式化输出结果而不进行评估。当您要将结果格式化为特定格式并将其提交到测试服务器时，它很有用。
-- `--metrics`: 用于评估 MMYOLO 中定义的模型的指标，如 COCO 标注格式的 "proposal" 。
+- `--metrics`: 用于评估 MMYOLO 中定义的模型的指标，如 COCO 标注格式的 "proposal"。
```

Comment:
does not have this argument. Pls. check the rest arguments 
https://github.com/open-mmlab/mmdeploy/blob/e47c6400b0f8bc1018bcc138b10c6d12bd5ceb26/tools/test.py#L14